jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/list/CaseInstanceListSearchViewImpl.java

```diff
@@ -93,27 +93,35 @@ public HTMLElement getElement() {
 
     @PostConstruct
     public void init() {
-        stream(CaseStatus.values()).forEach(s -> status.addOption(translationService.format(s.getLabel()), s.getName()));
+        stream(CaseStatus.values()).forEach(s -> status.addOption(translationService.format(s.getLabel()),
+                                                                  s.getName()));
         status.refresh();
 
-        stream(CaseInstanceSortBy.values()).collect(toMap(s -> s.name(), s -> translationService.format(s.name()))).entrySet().stream().sorted((e1, e2) -> e1.getValue().compareTo(e2.getValue())).forEach(s -> sortBy.addOption(s.getValue(), s.getKey()));
+        stream(CaseInstanceSortBy.values()).collect(toMap(s -> s.name(),
+                                                          s -> translationService.format(s.name()))).entrySet().stream().sorted((e1, e2) -> e1.getValue().compareTo(e2.getValue())).forEach(s -> sortBy.addOption(s.getValue(),
+                                                                                                                                                                                                                  s.getKey()));
```

Comment:
I tried to improve it a bit, but there is a lot going on there, hopefully, it is a bit more readable.